e2e/counter.spec.ts

```diff
@@ -9,12 +10,3 @@ test('counter', async ({ page }) => {
   await page.click('text=Increment');
   await expect(page.locator('text=Count: 3')).toBeVisible();
 });
-
-test('first render', async ({ browser }) => {
```

Comment:
I think that this is a bug. In dev mode, SSR is enforced, and no spinner. But in prod mode, there's a spinner

Reply:
> In dev mode, SSR is enforced

That's not expected. SSR will only be enables with `--with-ssr`.
Can you try it again? Maybe it's a timing issue.

Reply:
My mistake. Everything good